Equilobe.DailyReport.BL/BitBucket/BitBucketLogHelper.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Equilobe.DailyReport.Models;
+using Equilobe.DailyReport.Models.BitBucket;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Equilobe.DailyReport.BL.BitBucket
+{
+    public static class BitBucketLogHelper
+    {
+        public static Log LoadLog(List<Commit> commits, List<PullRequest> pullRequests, DateTime fromDate)
+        {
+            var log = new Log();
+
+            if (pullRequests != null)
```

Comment:
the code checks for null pullrequests, but you not for null commits.
i think it you can skip this check (as i see, there's no possibility to have null, just an empty list), and also use the object initializer to instantiate the log